udata/commands/db.py

```diff
@@ -134,18 +147,21 @@ def status():
 
 @m.option('-r', '--record', action='store_true',
           help='Only records the migrations')
-def migrate(record):
+@m.option('-d', '--dry-run', action='store_true', dest='dryrun',
```

Comment:
:)
This is a first attempt, a poor man dry run: It only print the request instead of executing it. But it's a start.